Makefile

```diff
@@ -15,7 +15,9 @@ TLP_SHCPL  ?= /usr/share/bash-completion/completions
 TLP_MAN    ?= /usr/share/man
 TLP_META   ?= /usr/share/metainfo
 TLP_RUN    ?= /run/tlp
+TLP_RUNCONF    ?= /run/tlp/tlp.conf
 TLP_VAR    ?= /var/lib/tlp
+TLP_CUSTOMIZE    ?= /etc/tlp.conf.d
```

Comment:
I'd prefer something shorter, like `TLP_CONFD`

Reply:
Below in the "Catenate DESTDIR to paths" section you'll also need `_RUNCONF` and `_CONFD`. Plus you'll have to create `TLP_CONFD` by the `install-tlp` target. Uninstalling by `uninstall-tlp` is not necessary.